cloud/virt.py

```diff
@@ -55,6 +55,26 @@
       - XML document used with the define command
     required: false
     default: null
+  xml_secure:
+    description:
+      - libvirt VIR_DOMAIN_XML_SECURE flag, only used for get_xml
+    required: false
+    default: 0
```

Comment:
default should read False (I know 0 gets passed in later, but here you are defining a boolean to the user)